### PR TITLE
Add auth:searchApiKeys

### PR DESCRIPTION
**Pull request by Yoann-Abbes**
## What does this PR do ?

Add auth:searchApiKeys API action https://github.com/kuzzleio/sdk-jvm/issues/29

### Other changes

Reverse wrong change on `build.gradle.kts`
Re-indent doc, and add comments

### CODE REVIEW

File: doc/1/controllers/auth/search-api-keys/index.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+---
+code: true
+type: page
+title: searchApiKeys
+description: Searches API keys for the currently loggued user.
+---
+
+# searchApiKeys
+
+<SinceBadge version="7.1.0" />
```
Comment (scottinet) [inline doc/1/controllers/auth/search-api-keys/index.md:10]:
```suggestion
<SinceBadge version="auto-version" />
```

File: doc/1/controllers/auth/search-api-keys/index.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+<SinceBadge version="Kuzzle 2.1.0" />
+
+Searches API keys for the currently loggued user.
```
Comment (scottinet) [inline doc/1/controllers/auth/search-api-keys/index.md:14]:
```suggestion
Searches API keys for the currently logged user.
```

File: doc/1/controllers/auth/search-api-keys/index.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+  Integer from
+  Integer size) throws NotConnectedException, InternalException
+```
+
+
+| Property | Type | Description |
+| --- | --- | --- |
+| `query` | <pre>ConcurrentHashMap<String, Object></pre> | Search query |
+| `from`     | <pre>Integer</pre><br/>(`0`)     | (optionnal) Offset of the first document to fetch   |
+| `size`     | <pre>Integer</pre>    | (optionnal) Maximum number of documents to retrieve per page     |
```
Comment (scottinet) [inline doc/1/controllers/auth/search-api-keys/index.md:40]:
```suggestion
| `from`     | <pre>Integer</pre><br/>(`0`)     | (optional) Offset of the first document to fetch   |
| `size`     | <pre>Integer</pre>    | (optional) Maximum number of documents to retrieve per page     |
```

File: doc/1/controllers/auth/search-api-keys/index.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+### query
+
+The search query to apply to API keys content, using [ElasticSearch Query DSL](https://www.elastic.co/guide/en/elasticsearch/reference/7.3/query-dsl.html) syntax.
```
Comment (scottinet) [inline doc/1/controllers/auth/search-api-keys/index.md:44]:
```suggestion
The search query to apply to API keys content, using the [ElasticSearch Query DSL](https://www.elastic.co/guide/en/elasticsearch/reference/7.3/query-dsl.html) syntax.
```

File: doc/1/controllers/auth/search-api-keys/index.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+
+If left empty, the result will return all available API keys of the currently loggued user.
```
Comment (scottinet) [inline doc/1/controllers/auth/search-api-keys/index.md:46]:
```suggestion
If left empty, the result will return all available API keys for the currently logged user.
```

File: doc/1/controllers/auth/search-api-keys/index.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+    from: Int = 0,
+    size: Int? = null): CompletableFuture<SearchResult>
+```
+
+
+| Property | Type | Description |
+| --- | --- | --- |
+| `query` | <pre>ConcurrentHashMap<String, Any?></pre> | Search query |
+| `from`     | <pre>Int</pre><br/>(`0`)     | (optionnal) Offset of the first document to fetch   |
+| `size`     | <pre>Int</pre>    | (optionnal) Maximum number of documents to retrieve per page     |
```
Comment (scottinet) [inline doc/1/controllers/auth/search-api-keys/index.md:74]:
```suggestion
| `from`     | <pre>Int</pre><br/>(`0`)     | (optional) Offset of the first document to fetch   |
| `size`     | <pre>Int</pre>    | (optional) Maximum number of documents to retrieve per page     |
```

File: doc/1/controllers/auth/search-api-keys/index.md [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+
+### query
+
+The search query to apply to API keys content, using [ElasticSearch Query DSL](https://www.elastic.co/guide/en/elasticsearch/reference/7.3/query-dsl.html) syntax.
+
+If left empty, the result will return all available API keys of the currently loggued user.
```
Comment (scottinet) [inline doc/1/controllers/auth/search-api-keys/index.md:80]:
```suggestion
The search query to apply to API keys content, using the [ElasticSearch Query DSL](https://www.elastic.co/guide/en/elasticsearch/reference/7.3/query-dsl.html) syntax.

If left empty, the result will return all available API keys of the currently logged user.
```

**Review by scottinet — CHANGES_REQUESTED** [review]
Documentation fixes (sorry I misclicked on "comment" instead of "request changes")

**Review by scottinet — APPROVED** [review]

File: build.gradle.kts [hunk 8]
```diff
@@ -156,4 +156,5 @@ tasks.withType<Jar> {
             )
         )
     }
+    from(configurations.compileClasspath.get().map { if (it.isDirectory()) it else zipTree(it) })
```
Comment (jenow) [inline build.gradle.kts:159]: lol.

**Review by jenow — APPROVED** [review]

**Pull request merged**